client/me/security-checkup/account-recovery-email.jsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getAccountRecoveryEmail,
+	isAccountRecoveryEmailActionInProgress,
+	isAccountRecoveryEmailValidated,
+} from 'state/account-recovery/settings/selectors';
+import { getOKIcon, getWarningIcon } from './icons.js';
+import QueryAccountRecoverySettings from 'components/data/query-account-recovery-settings';
+import SecurityCheckupNavigationItem from './navigation-item';
+
+class SecurityCheckupAccountRecoveryEmail extends React.Component {
+	static propTypes = {
+		accountRecoveryEmail: PropTypes.oneOfType( [ PropTypes.bool, PropTypes.string ] ),
+		accountRecoveryEmailActionInProgress: PropTypes.bool,
+		accountRecoveryEmailValidated: PropTypes.bool,
+		translate: PropTypes.func.isRequired,
+	};
+
+	render() {
+		const {
+			accountRecoveryEmail,
+			accountRecoveryEmailActionInProgress,
+			accountRecoveryEmailValidated,
+			translate,
+		} = this.props;
+
+		if ( accountRecoveryEmailActionInProgress ) {
+			return (
+				<React.Fragment>
+					<QueryAccountRecoverySettings />
+					<SecurityCheckupNavigationItem isPlaceholder={ true } />;
+				</React.Fragment>
+			);
+		}
+
+		let icon, description;
+
+		if ( ! accountRecoveryEmail ) {
+			icon = getWarningIcon();
+			description = translate( 'You do not have a recovery email address.' );
+		} else if ( ! accountRecoveryEmailValidated ) {
+			icon = getWarningIcon();
+			description = translate(
+				'You still need to verify your recovery email address: {{strong}}%(recoveryEmailAddress)s{{/strong}}',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **10** times:
`translate( 'You still need to validate your recovery email address: {{strong}}%(recoveryEmailAddress)s{{/strong}}' )` *ES Score: 7*

Reply:
The switch to `verify` was intentional.